plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -463,7 +463,7 @@ private TestExecutionResult testPluginAgainst(MavenCoordinates coreCoordinates,
             while (entries.hasMoreElements()) {
                 JarEntry entry = entries.nextElement();
                 String name = entry.getName();
-                Matcher m = Pattern.compile("WEB-INF/lib/jenkins-core-([0-9.]+(?:-[0-9.]+)?(?:-SNAPSHOT)?)[.]jar").matcher(name);
+                Matcher m = Pattern.compile("WEB-INF/lib/jenkins-core-([0-9.]+(?:-[0-9.]+)?(?:-(alpha|beta|RC|rc)-[0-9.]+)?(?:-SNAPSHOT)?)[.]jar").matcher(name);
```

Comment:
@raul-arabaolaza not since [maven3](https://github.com/apache/maven/blob/eaa4cdfb8e2f32306b425ecad68fde25f21271fa/maven-artifact/src/main/java/org/apache/maven/artifact/versioning/ComparableVersion.java#L357)...

The other cases are also special in maven :)

https://maven.apache.org/ref/3.3.9/maven-artifact/apidocs/index.html?org/apache/maven/artifact/versioning/ComparableVersion.html
```
strings are checked for well-known qualifiers and the qualifier ordering is used for version ordering. Well-known qualifiers (case insensitive) are:
alpha or a
beta or b
milestone or m
rc or cr
snapshot
(the empty string) or ga
```